features/step_definitions/generic_steps.rb

```diff
@@ -2,6 +2,10 @@
   unselect(item, :from => selector)
 end
 
+When /^(?:|I )pry in to the system$/ do
+  require 'pry'
+  binding.pry
```

Comment:
Just wondering -- can you explain the difference between this step and the one below it, which is:
```
Then /^debug$/ do
    binding.pry
end
```
Basically, why do we need one with `require 'pry'` and one without?

Also, I think you need a blank line between these two steps.